contrib/babelfishpg_tsql/sql/sys_functions.sql

```diff
@@ -1232,6 +1232,46 @@ $$
 STRICT
 LANGUAGE plpgsql IMMUTABLE;
 
+CREATE OR REPLACE FUNCTION sys.daydiff_between_dates(IN d1 INTEGER, IN m1 INTEGER, IN y1 INTEGER, IN d2 INTEGER, IN m2 INTEGER, IN y2 INTEGER) RETURNS INTEGER AS $$
```

Comment:
better would be to have a function which returns number of days and just call it two time res = func1() - func2()

Reply:
modified